libopenage/engine.cpp

```diff
@@ -548,4 +542,12 @@ void Engine::move_phys_camera(float x, float y, float amount) {
 	this->engine_coord_data->camgame_phys += cam_delta.to_phys3();
 }
 
+void Engine::start_game(std::unique_ptr<GameMain> game) {
+	// TODO: maybe implement a proper 1-to-1 connection
+	assert(game);
```

Comment:
So far we have `util::Error` is handled exactly like an assert, it quits the program, but allows backtrace collection and error messages. We don't have an exception hierarchy yet, so all exceptions are fatal and we only used them for that. So I'd say there's no need for any `assert` and we can use `util::Error`.